deluge/ui/web/server.py

```diff
@@ -536,6 +585,28 @@ def __init__(self):
             theme = CONFIG_DEFAULTS.get('theme')
         self.__stylesheets.insert(1, 'themes/css/xtheme-%s.css' % theme)
 
+    def setup_webapidoc(self, enabled=True):
+        """
+        Enable or disable the Webapidoc resource
+
+        Args:
+            enabled (bool): Enable or disable Webapidoc
+
+        """
+        path = 'swagger-ui'
+
+        if enabled:
+            if self.js.has_script(path):
+                return
+
+            self.putChild(self.webapidoc_path.encode(), self.webapidoc_resource)
+            self.js.add_script_folder(path, rpath('js', path))
+        else:
```

Comment:
Is this to handle users switching the config from enable to disable?

Reply:
Yes, it is. Would be easier to just have it always enabled